### PR TITLE
Evasion no longer lets you slash marines

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
https://i.imgur.com/LmHmI2l.png

Title basically.
## Why It's Good For The Game
Makes runner evasion a pretty dedicated escape tool from group rather than just god mode.

## Changelog
:cl:
balance: Runner can no longer slash in invasion.
/:cl:

### CODE REVIEW

**Comment by Zangeiti** [comment]
SPEEDMERG NOW!!!

**Comment by lKiseki** [comment]
but why tho

**Comment by Aroliacue** [comment]
This is definitely a needed change but I also hope we start seeing some xeno buffs soon too!

**Comment by SimplySmiley123** [comment]
Finally I'll be able to help a teammate fend off a rouney without setting both on fire first.

**Comment by Naaanii** [comment]
Who let him cook?

**Comment by chizzy376** [comment]
SPEEDMERGE!!!!
Thank you lord MJP for bringing us one of the greatest push requests known to man

**Comment by Al-1ce** [comment]
Rouny doesn't deserve this.

**Comment by Ragecrabbo** [comment]
Rouny deserves this for his crimes

**Comment by Oscari-pog** [comment]
Hold up, let him cook

**Comment by Arturlang** [comment]
Keep in mind it's not a godmode, there are counters to this, PBs, flamers, plasma pistols, meele, how is this godmode?

File: code/modules/mob/living/carbon/xenomorph/castes/runner/runner.dm [hunk 1]
```diff
@@ -28,6 +28,14 @@
 	if(. == CONSCIOUS && layer != initial(layer))
 		layer = MOB_LAYER
 
+/mob/living/carbon/xenomorph/runner/UnarmedAttack(atom/A, has_proximity, modifiers)
+	/// Runner should not be able to slash while evading.
```
Comment (Arturlang) [inline code/modules/mob/living/carbon/xenomorph/castes/runner/runner.dm:32]: There should be a trait you can use to block hand actions

**Comment by Kuro020** [comment]
based

**Comment by Woker2004** [comment]
I will state that the chunk of runner's damage came from this, although the runner evasion could already be counter by tanglefoot, fire, and getting pointblanked, alongside with smoke making xenos unable to see I honestly see this PR being just a cope PR because you have the tools to deal with a evading rouny

**Comment by SimplySmiley123** [comment]
> Keep in mind it's not a godmode, there are counters to this, PBs, flamers, plasma pistols, meele, how is this godmode?

Because trying to hit a fast target with those is a once in a blue moon, even better if you don't wanna set someone on your own team on fire.

And also trying to melee a rouney who can just knock you down then dance around you like crazy, yeah totally a good idea.

**Comment by AlexCrimson** [comment]
What is it with people constantly trying to nerf Runners? Or just Xenos in general. Evasion has so many counters already. Even without them, a full combo does not even come close to killing a Marine. Runners are already laughably fragile. Evasion is the only thing that lets them contribute against groups of Marines.

**Comment by Ro2tCrab** [comment]
uh....

Marine Winrate - 61%
Xenomorph Winrate - 39%

Another nerf... cool.

**Comment by yyzsong** [comment]
Rouny is fine already.

**Comment by Loafin34** [comment]
Rouny's evasion is stupid in general, but I don't see people using it while slashing marines. It's usually the pounce, slash hand two-three times, then pop evasion and get out.

**Comment by Lumipharon** [comment]
Lets see how it goes.

**Comment by lKiseki** [comment]
you really just merged this huh

**Comment by novaepee** [comment]
SELF MERGE BE LIKE

**Comment by Al-1ce** [comment]
Community driven.

**Comment by Al-1ce** [comment]
Something tells me this isn’t going to be a temporary experiment.

**Comment by BonkaiB** [comment]
self merge point at the goober

**Comment by AlexCrimson** [comment]
Well look at that it was not just a test run

**Pull request merged**